tools/haddock-score.py

```diff
@@ -0,0 +1,221 @@
+"""
+A simple tool to calculate the HADDOCK-score of a complex.
+
+Usage:
+    python haddock-score complex.pdb
+"""
+
+import argparse
+import subprocess
+from pathlib import Path
+
+from pdbtools.pdb_tidy import tidy_pdbfile
+
+from haddock import toppar_path as TOPPAR_DIR
+from haddock.core.defaults import cns_exec as CNS_EXEC
+from haddock.gear.haddockmodel import HaddockModel
+from haddock.gear.yaml2cfg import read_from_yaml_config
+from haddock.libs.libcns import prepare_cns_input
+from haddock.libs.libontology import PDBFile, TopologyFile
+from haddock.modules.scoring.emscoring import DEFAULT_CONFIG as SCORING_DEFAULT_CONFIG
+from haddock.modules.scoring.emscoring import RECIPE_PATH as emscoring_module_folder
+from haddock.modules.topology.topoaa import DEFAULT_CONFIG as TOPO_DEFAULT_CONFIG
+from haddock.modules.topology.topoaa import RECIPE_PATH as topoaa_module_folder
+from haddock.modules.topology.topoaa import generate_topology
+
+
+def topo_wrapper(pdb):
+    """
+    Wrapper for the topology generation.
+
+    Parameters
+    ----------
+    pdb : str
+        Path of the PDB file.
+
+    Returns
+    -------
+    pdb_object : PDBFile
+        The PDBFile object.
+    """
+    model_path = Path(pdb).resolve()
+
+    tidy_model_path = Path(f"{model_path.stem}_tidy.pdb").resolve()
+    with open(model_path, "r") as inp_fh:
+        with open(tidy_model_path, "w") as out_fh:
+            for line in tidy_pdbfile(inp_fh):
+                out_fh.write(line)
```

Comment:
Single call to save to disk

```suggestion
            lines = list(tidy_pdbfile(inp_fh))
                out_fh.write(os.linesep.join(lines))
```